biz.aQute.bndlib/src/aQute/bnd/build/model/BndEditModel.java

```diff
@@ -967,14 +969,67 @@ public void setTestSuites(List<String> suites) {
 	}
 
 	public List<HeaderClause> getPlugins() {
-		return doGetObject(Constants.PLUGIN, headerClauseListConverter);
+		// return all plugins
```

Comment:
Not sure if this works with inheritance of properties & bnd file properties? It is a powerful but nasty problem
   
     file 1->* 'plugin.*' property 0->* clause 

To make this work I think you need to model this as an object. When you edit a file, you can edit the local ones in the BndEditoModel but I think the inherited ones should be visible greyed out?